source/includes/fundamentals/code-snippets/CRUD/bulkOps.go

```diff
@@ -87,13 +90,15 @@ func main() {
 
 	fmt.Println("\nReplaceOneModel:\n")
 	{
+		// Creates a write model that specifies a replace operation
```

Comment:
S: similar suggestion as above; applies to the rest of the file as well
```suggestion
		// Creates instructions to replace the first document that matches the query
```